samples/client/petstore/java/feign/src/test/java/org/openapitools/client/api/AnotherFakeApiTest.java

```diff
@@ -1,9 +1,10 @@
 package org.openapitools.client.api;
+//TODO update packages 
```

Comment:
@hugo-ma-alves thanks for the PR, which looks good overall.

I saw the TODO here. What needs to be done?

Reply:
It was a reminder to remove the unused imports. All done now